samples/WebEndpoints/WebEndpoint/Endpoints/WebEndpoint.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using ProconTel.Sdk.Attributes;
+using ProconTel.Sdk.Builders;
+using ProconTel.Sdk.Communications.Middlewares;
+using ProconTel.Sdk.Messages;
+using ProconTel.Sdk.Services;
+using ProconTel.Sdk.StandardEndpoints;
+
+namespace WebEndpoints.WebApiEndpoint.Endpoints
+{
+  [EndpointMetadata(Name = "[Test] WebServer", SupportedRoles = SupportedRoles.Both)]
+  [SupportsXmlProtocol]
+  public class WebEndpoint : IEndpointLifeTimeCycle, IHandler
```

Comment:
This endpoint doesn't have to implement `IHandler`

Reply:
Removed